static/app/utils/formatters.tsx

```diff
@@ -181,7 +181,12 @@ export function getDuration(
  *
  * e.g. 1 hour 25 minutes 15 seconds
  */
-export function getExactDuration(seconds: number, abbreviation: boolean = false) {
+export function getExactDuration(
```

Comment:
Let's move this out into it's own PR as well, I've got some feedback for this one as well I think